src/components/common/YourCompany/NameTranslations/ListNameTranslation.vue

```diff
@@ -1,118 +1,134 @@
-              <v-menu offset-y left nudge-bottom="4">
-                <template v-slot:activator="{ on }">
-                  <v-btn
-                    text
-                    small
-                    v-on="on"
-                    color="primary"
-                    class="actions__more-actions__btn"
-                    :disabled="isAddingNameTranslation">
-                    <v-icon>mdi-menu-down</v-icon>
-                  </v-btn>
-                </template>
-                <v-list class="actions__more-actions">
-                  <v-list-item  @click="emitNameEdit(index)">
-                    <v-list-item-subtitle>
-                      <v-icon small>mdi-pencil</v-icon>
-                      <span class="ml-1">Correct</span>
-                    </v-list-item-subtitle>
-                  </v-list-item>
-                  <v-list-item  @click="emitRemoveName(index)">
-                    <v-list-item-subtitle>
-                      <v-icon small>mdi-delete</v-icon>
-                      <span class="ml-1">Remove</span>
-                    </v-list-item-subtitle>
-                  </v-list-item>
-                </v-list>
-              </v-menu>
-            </span>
-          </div>
-        </v-col>
-        <v-col v-else>
-          <div class="actions">
-            <span class="edit-action">
+        <v-chip v-if="translation.action === ActionTypes.REMOVED"
+          x-small label color="grey lighten-2" text-color="grey darken-4"
+        >
+          <span>REMOVED</span>
+        </v-chip>
+      </v-col>
+
+      <!-- Actions Column - Edited or Removed-->
+      <v-col v-if="translation.action === ActionTypes.EDITED || translation.action === ActionTypes.REMOVED">
+        <!--  -->
+        <div class="actions mt-n1 float-right">
+          <span class="edit-action">
+            <v-btn
+              text small
+              color="primary"
+              :disabled="isAddingNameTranslation"
+              @click="undoTranslation(index)"
+            >
+              <v-icon small>mdi-undo</v-icon>
+              <span>Undo</span>
+            </v-btn>
+          </span>
+
+          <!-- more actions menu -->
+          <template v-if="translation.action !== ActionTypes.REMOVED">
+            <v-menu offset-y>
+              <template v-slot:activator="{ on }">
+                <v-btn
+                  text color="primary"
+                  class="more-actions-btn"
+                  v-on="on"
+                  :disabled="isAddingNameTranslation"
+                >
+                  <v-icon>mdi-menu-down</v-icon>
+                </v-btn>
+              </template>
+
+              <v-list class="more-actions-list">
+                <v-list-item @click="editTranslation(index)">
+                  <v-list-item-title>
+                    <v-icon small>mdi-pencil</v-icon>
+                    <span class="ml-2">Correct</span>
+                  </v-list-item-title>
+                </v-list-item>
```

Comment:
This change to the buttons/menus (and same change below) is to be consistent with Create UI.

Before:

![image](https://user-images.githubusercontent.com/10002889/202012337-078baff4-6f39-41d4-bf4d-784766d71a42.png)

After:

![image](https://user-images.githubusercontent.com/10002889/202012399-17bf7393-842a-49d7-b3a1-e61f72d62895.png)

Reply:
In Create UI:

![image](https://user-images.githubusercontent.com/10002889/202012661-cce5c9c2-fff1-4d9f-ad3f-d0abac18fdae.png)

Reply:
Good catch. Yes! I will have another PR for Create UI with some updates to back-port from this PR (Edit UI).